app/templates/app.js

```diff
@@ -6,16 +6,16 @@ var Message = require('azure-iot-device').Message;
 
 Client.fromEnvironment(Transport, function (err, client) {
   if (err) {
-    console.log('error:' + err);
+    throw err;
```

Comment:
we can still keep the console error for user to docker logs fetch the output

Reply:
I think users will get console errors for free with uncaught exception.